src/mol-util/array.ts

```diff
@@ -105,6 +105,21 @@ export function arraySetRemove<T>(xs: T[], x: T) {
     return true;
 }
 
+export function arrayAreIntersecting<T>(xs: T[], ys: T[]) {
+    for (let i = 0, il = xs.length; i < il; ++i) {
+        if (ys.includes(xs[i])) return true;
+    }
+    return false;
+}
+
+export function arrayIntersectionSize<T>(xs: T[], ys: T[]) {
+    let count = 0;
+    for (let i = 0, il = xs.length; i < il; ++i) {
+        if (ys.includes(xs[i])) count += 1;
+    }
+    return count;
+}
```

Comment:
I am a little bit worried about the ``O(n^2)`` complexity here. It should be fine for the rings as they are small, but would be careful about leaving the method here as it is. Maybe add a comment at least saying it's for small arrays only?

Reply:
Good point, will add a note